c2corg_api/views/forum.py

```diff
@@ -102,9 +109,39 @@ def collection_post(self):
             raise HTTPInternalServerError('Error with Discourse')
 
         if "topic_id" in response:
-            document_topic = DocumentTopic(topic_id=response['topic_id'])
+            topic_id = response['topic_id']
+
+            document_topic = DocumentTopic(topic_id=topic_id)
             locale.document_topic = document_topic
             update_cache_version_direct(locale.document_id)
             DBSession.flush()
 
+            if locale.type == document_types.OUTING_TYPE:
+                self.invite_participants(client, locale, topic_id)
+
         return response
+
+    def invite_participants(self, client, locale, topic_id):
+        try:
```

Comment:
You are using two `try ... except`, the first one to avoid that calling `self.invite_participants` makes the request fail, and the 2nd to invite the other participants if the invitation for one participant fails?

I am ok with the 2nd `try ... except`. But I think it would make more sense to put the first `try ... except` around the call to `self.invite_participants(client, locale, topic_id)`.